README.md

```diff
@@ -1557,5 +1557,6 @@ Table of Contents
 HTML Themes and UI Kits to create your applications faster!
   * [Web.Dev](https://web.dev/measure/) — This is a free tool that allows you to see the performance of your website and improve the SEO to get higher rank list in search engines.
   * [SmallDev.tools](https://smalldev.tools/) — A free tool for developers that allows you to Encode/Decode various formats, Minify HTML/CSS/Javascript, Beautify, Generate Fake/Testing real like dataset in JSON/CSV & multiple other formats and many more features. With a delightful interface.
+  * [Openbase.com](https://openbase.com/) - Compare open-source packages with powerful metrics and user reviews.
```

Comment:
useful I guess, but dont think it really fits the purpose of this particular list sorry.